CHANGELOG.md

```diff
@@ -1,5 +1,206 @@
 # Unreleased
 
+## Unreleased 9.0.0 (Breaking release)
```

Comment:
I think this should be

```suggestion
# 9.0.0 (Breaking release)
```

Leaving the 'Unreleased' section empty (ready for entries for the next version)

That'll also mean other heading levels need adjusting throughout.

Reply:
Ah you're right! Will fix :)